core/src/main/resources/db/migration/V3.1__Remove_V1_Concepts.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS feature_sets CASCADE;
```

Comment:
Can we postpone table removal? Maybe move it to separate PR and not merge for now